event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamObject.java

```diff
@@ -3,15 +3,17 @@
 import java.time.ZonedDateTime;
 import java.util.UUID;
 
-public class EventStream {
+public class EventStreamObject {
```

Comment:
EventStream Object is better than EventStream?

Reply:
Yes as we have an interface called EventStream and it very confusing to have Interface and Objects with the same name